gf-webapp-admin/src/component/registration/Registration.module.scss

```diff
@@ -28,7 +28,7 @@
 }
 
 .registrationBtn {
-  padding: 5px 10px;
+  padding: 5px 20px;
```

Comment:
it is desirable to put in constants paddings, marging etc

Reply:
If it is incapsulated paddings, margin etc. you shouldn't put them in global CSS variables